artifacts/artifact_funcs.jl

```diff
@@ -1,15 +1,8 @@
 import ArtifactWrappers as AW
+import ClimaUtilities.ClimaArtifacts: @clima_artifact
 
-function co2_dataset_path()
-    co2_dataset = AW.ArtifactWrapper(
-        @__DIR__,
-        "co2",
-        AW.ArtifactFile[AW.ArtifactFile(
-            url = "https://caltech.box.com/shared/static/xg028wnsn57wam6euwrh98fe43ibei8g.nc",
-            filename = "mauna_loa_co2.nc",
-        ),],
-    )
-    return AW.get_data_folder(co2_dataset)
+function co2_dataset_path(; context = nothing)
```

Comment:
maybe we can just call `@clima_artifact` directly instead of `co2_dataset_path`, like we do for SST and SIC [here](https://github.com/CliMA/ClimaCoupler.jl/blob/6d022101262c2c93c572979b7562dcc929bfc2ab/experiments/ClimaEarth/run_amip.jl#L184)

Reply:
you can also delete this function once you make this change